.idea/runConfigurations/Run_Docs_Webiste_Locally.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run Docs Webiste Locally" type="ShConfigurationType">
+    <option name="SCRIPT_TEXT" value="unzip -o webHelpD2-all.zip -d webHelpD2-all &amp;&amp; cd webHelpD2-all &amp;&amp; python3 -m http.server 8000" />
```

Comment:
also, probably add all created webhelp files to .gitignore to prevent accidental additions